Pipfile

```diff
@@ -50,6 +50,7 @@ blinker = "*"
 humanize = "*"
 flask-talisman = "*"
 marshmallow = "*"
+pytest = "*"
```

Comment:
Why are we duplicating pytest?  Shouldn't it only be there as part of the dev packages?

Reply:
Good catch, that was not intended! I've re-pushed